Hospital-System/HospitalSystem.BLL/Services/Appointment/AppointmentService.cs

```diff
@@ -0,0 +1,64 @@
+﻿using AutoMapper;
+using HospitalSystem.BLL.Services.Appointment.Interfaces;
+using HospitalSystem.Common.Models.DTOs.Appointment;
+using HospitalSystem.DAL;
+using HospitalSystem.DAL.Entities;
+using HospitalSystem.DAL.Repository.Interfaces;
+using Microsoft.EntityFrameworkCore;
+
+namespace HospitalSystem.BLL.Services.Appointment;
+
+public class AppointmentService : IAppointmentService
+{
+    private readonly IRepository<DAL.Entities.Appointment> _repositoryAppointment;
+    private readonly IRepository<DoctorTimeSlot> _repositoryDoctorTimeSlot;
+    private readonly IRepository<DAL.Entities.Doctor> _repositoryDoctor;
+    private readonly IRepository<Patient> _repositoryPatient;
```

Comment:
This variables is not used.
![image](https://github.com/doomer2004/Hospital-system./assets/67170413/80cce552-74d0-498b-a64d-0abfe5f0f318)